test/utils/describeConformanceV5.js

```diff
@@ -251,14 +245,18 @@ const fullSuite = {
  */
 export default function describeConformanceV5(minimalElement, getOptions) {
   const { after: runAfterHook = () => {}, only = Object.keys(fullSuite), skip = [] } = getOptions();
+
+  const render = createClientRender();
```

Comment:
The intent is double. Remove the amount of work done in the tests. And fix the cleanup issue when using mocking the time.

Reply:
> The intent is double. 

What does this mean?

> And fix the cleanup issue when using mocking the time.

What is "the issue"?

Reply:
I will open a PR that showcases it. We don't have to solve it here anymore since we gave up on the setTimeout approach

Reply:
So we can revert the change for now?

Reply:
Yes

Reply:
https://github.com/mui-org/material-ui/pull/25003